.gitignore

```diff
@@ -11,6 +11,7 @@ blobs
 .DS_Store
 .final_builds/jobs/**/*.tgz
 .final_builds/packages/**/*.tgz
+.byebug_history
```

Comment:
This should go in your global gitignore rather than in this project specific one.